src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx

```diff
@@ -202,7 +202,7 @@ export default class ResultTable extends React.Component {
 
 const GridContainer = styled(({visibleRows, ...props}) => <Panel {...props} />)`
   height: ${p =>
-    p.visibleRows * 40 +
+    p.visibleRows * 41 +
```

Comment:
Can we extract these magic numbers into constants?

Reply:
done